<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="plus_button">
      <property name="text">
       <string>plus</string>
      </property>
      <property name="icon">
       <iconset resource="resources_icon.qrc">
        <normaloff>:/newPrefix/plus.png</normaloff>:/newPrefix/plus.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox"/>
    </item>
    <item>
     <widget class="QPushButton" name="minus_button">
      <property name="text">
       <string>minus</string>
      </property>
      <property name="icon">
       <iconset resource="resources_icon.qrc">
        <normaloff>:/newPrefix/plus.png</normaloff>:/newPrefix/plus.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources_icon.qrc"/>
 </resources>
 <connections/>
</ui>
